src/Controller/AdminRegistrationController.php

```diff
@@ -30,17 +34,28 @@ class AdminRegistrationController extends LeftAndMain
     private static $url_handlers = [
         'GET register/$Method' => 'startRegistration',
         'POST register/$Method' => 'finishRegistration',
-        'GET remove/$Method' => 'removeRegisteredMethod',
+        'DELETE remove/$Method' => 'removeRegisteredMethod',
```

Comment:
`DELETE remove/:identifier` is kind of doubling up the directive - a more RESTful endpoint would be `DELETE method/:identifier`. Maybe we should adjust these if we get time.

Reply:
Good idea. I don't think the registration handlers should change, but this one should for sure. I don't think there's an obvious restful way to combine "set default method" with `[httpMethod] method/$Method` either, so the result would probably be `DELETE method/$Method` only for now?

Reply:
In RESTful terms, setting the default method could potentially come in the form of an update to the method itself (`POST method/$Method`, with something like `{default: true}` in the body). You're right in that this would require significant changes to how that endpoint is currently implemented, so alternatively you could use `PUT` to designate an update over a creation and split the logic that way.

Another RESTish option would be to treat 'default' as a 'sub-resource' of the method - so `PUT method/$Method/default`.

Reply:
I like that last option

Reply:
Although semantically you’re updating the member not the method, since we store the default flag on the member

Reply:
But you said sub-resource, so 👍